### PR TITLE
BETA: Register minerwolfybt.is-a.dev

### PR DESCRIPTION
**Pull request by nquangdung**
Added `minerwolfybt.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by PinkLittleKitty** [comment]
Vercel is not allowed

File: domains/_vercel.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "bebapbapbeo1310",
+           "email": "ngdung853@gmail.com",
+           "discord": "947884053544189992"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=minerwolf.is-a.dev,f25676adb5c75d453115"
```
Comment (Stef-00012) [inline domains/_vercel.json:9]: Vercel is not supported

File: domains/minerwolf.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "bebapbapbeo1310",
+           "email": "ngdung853@gmail.com",
+           "discord": "947884053544189992"
+        },
+    
+        "record": {
+            "TXT": "vc-domain-verify=minerwolf.is-a.dev,f25676adb5c75d453115"
```
Comment (Stef-00012) [inline domains/minerwolf.json:9]: Vercel is not supported

File: domains/minerwolfybt.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "bebapbapbeo1310",
+        "email": "ngdung853@gmail.com"
+    },
+    "record": {
+        "CNAME": "vc-domain-verify=minerwolf.is-a.dev,f25676adb5c75d453115"
```
Comment (Stef-00012) [inline domains/minerwolfybt.json:7]: Vercel is not supported

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Pull request closed without merging**